pipelines/broad/dna_seq/germline/joint_genotyping/test_inputs/Scientific/bge.json

```diff
@@ -0,0 +1,36 @@
+{
+  "JointGenotyping.sample_name_map": "FILL_IN",
+  "JointGenotyping.callset_name": "bge_joint_genotyping_plumbing",
+  "JointGenotyping.run_vets": true,
+
+  "JointGenotyping.unpadded_intervals_file": "",
```

Comment:
Were `unpadded_intervals_file`, `eval_interval_list`, and `targets_interval_list` intentionally left empty for now?